addons/calendar/data/mail_template_data.xml

```diff
@@ -75,7 +75,7 @@
             <ul>
                 <t t-if="object.event_id.location">
                     <li>Location: <t t-out="object.event_id.location or ''">Bruxelles</t>
-                        (<a target="_blank" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.event_id.location}}">View Map</a>)
+                        <a target="_blank" t-if="not is_online or is_online and object.event_id.location != object.event_id.appointment_type_id.location_id.name" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.event_id.location}}">(View Map)</a>
```

Comment:
Note that in base calendar module, appointment_type_id field does not exist so I bet you are going to encounter an issue. You should be defensive in template and check 'appointment_type_id' if in object.event_id

Reply:
Indeed `appointment_type_id` does not exist in calendar module. Hence I added the `is_online` condition with an `and` operator before checking `appointment_type_id`.

The following code already exists in the file for checking this condition. Hence I reused the variable.
```
<t t-set="is_online" t-value="'appointment_type_id' in object.event_id and object.event_id.appointment_type_id"/>
```

Reply:
Arf indeed sorry for that :) forgot to check the is_online condition. Note that in calendar_template_meeting_invitation is_online does not seem defined ?